ci/codespell.ignores

```diff
@@ -23,3 +23,8 @@ ether
 expport
 coo
 spawnve
+inflight
+warmup
+ro
+nam
+clen
```

Comment:
Are these actually valid?

Reply:
I noticed this too, seems valid in a way, but could cause "nam" vs "name" problems to slip in later. 
/src/common/tests/compress_tests.c in the "Text to be compressed" origin_buf[] array.

Reply:
I would think that file/array should instead have an exception, since that isn't english

Reply:
I changed the name of clen.   Added artifac (which is apparently something artifactory uses).   I don't see how to not do Nam though since it's part of that buffer of Latin text in src/common/tests/compress_tests.c without modifying the test.  Also added resizeable because it's an alternative spelling per google even if it's not the correct one.   It's in the dictionary and it didn't feel right to remove it.

Note codespell isn't perfect.

Reply:
resizeable is an alternate UK spelling, so it depends on whether we care about that

Reply:
@ashleypittman told me to remove it so I'm gonna take that as authoritative

Reply:
Pylint will use US spellings and codespell will use it's GB-to-US wordlist.  It's not ideal from my perspective obviously but I'll live with it.

https://github.com/daos-stack/daos/blob/master/utils/cq/daos_pylint.py#L561
https://github.com/daos-stack/daos/blob/master/.github/workflows/spelling.yml#L18